docs/public-networks/how-to/bonsai-limit-trie-logs.md

```diff
@@ -0,0 +1,191 @@
+---
+title: Reduce database size using Bonsai Tries
+sidebar_position: 12
+description: Enable this feature to reduce the size of your database
+tags:
+  - public networks
+---
+
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+To decrease the database size using the [Bonsai Trie](../concepts/data-storage-formats#bonsai-tries) data storage policy in Besu, enable the early access feature `--Xbonsai-limit-trie-logs-enabled`. 
+When enabled, this feature can reduce database growth by more than 3GB each week on mainnet.
+
+## Limit Trie Logs for Bonsai
```

Comment:
```suggestion
## Step-by-Step Guide
```

Reply:
We don't usually add headings that are not specific or descriptive (probably for SEO and findability reasons). The docs usually have scannable headings (that start with an action word). Is it okay if we add a more descriptive title for the procedure?

Reply:
"Limit trie logs" is just the first part of the guide, the rest of the guide is instructing the user how to prune old trie logs...I think "Limit trie logs and prune" is more accurate.